src/Examples/UnitModels/Operations/skeleton_unit_testing.ipynb

```diff
@@ -0,0 +1,890 @@
+    "m.fs.pervap.temperature_in = Var(m.fs.time, initialize=298.15, units=pyunits.K)\n",
+    "m.fs.pervap.pressure_in = Var(m.fs.time, initialize=101e3, units=pyunits.Pa)\n",
+    "\n",
+    "# output vars for skeleton\n",
+    "m.fs.pervap.perm_flow = Var(m.fs.time, m.fs.pervap.phase_list, m.fs.pervap.comp_list,\n",
+    "                                      initialize=1.0, units=pyunits.mol/pyunits.s)\n",
+    "m.fs.pervap.ret_flow = Var(m.fs.time, m.fs.pervap.phase_list, m.fs.pervap.comp_list,\n",
+    "                                      initialize=1.0, units=pyunits.mol/pyunits.s)\n",
+    "m.fs.pervap.temperature_out = Var(m.fs.time, initialize=298.15, units=pyunits.K)\n",
+    "m.fs.pervap.pressure_out = Var(m.fs.time, initialize=101e3, units=pyunits.Pa)\n",
+    "m.fs.pervap.vacuum = Var(m.fs.time, initialize=1.3e3, units=pyunits.Pa)\n",
+    "\n",
+    "# dictionaries relating state properties to custom variables\n",
+    "inlet_dict = {\"flow_mol_phase_comp\": m.fs.pervap.flow_in,\n",
+    "              \"temperature\": m.fs.pervap.temperature_in,\n",
+    "              \"pressure\": m.fs.pervap.pressure_in}\n",
+    "retentate_dict = {\"flow_mol_phase_comp\": m.fs.pervap.ret_flow,\n",
+    "                  \"temperature\": m.fs.pervap.temperature_out,\n",
+    "                  \"pressure\": m.fs.pervap.pressure_out}\n",
+    "permeate_dict = {\"flow_mol_phase_comp\": m.fs.pervap.perm_flow,\n",
+    "                 \"temperature\": m.fs.pervap.temperature_out,\n",
+    "                 \"pressure\": m.fs.pervap.vacuum}\n",
+    "\n",
+    "m.fs.pervap.add_ports(name=\"inlet\", member_dict=inlet_dict)\n",
+    "m.fs.pervap.add_ports(name=\"retentate\", member_dict=retentate_dict)\n",
+    "m.fs.pervap.add_ports(name=\"permeate\", member_dict=permeate_dict)\n",
+    "\n",
+    "# internal vars for skeleton\n",
+    "energy_activation_dict = {(0, 'Liq', 'water'): 51e3, (0, 'Liq', 'ethylene_glycol'): 53e3}\n",
+    "m.fs.pervap.energy_activation = Var(m.fs.time, m.fs.pervap.phase_list, m.fs.pervap.comp_list,\n",
+    "                                    initialize=energy_activation_dict, units=pyunits.J/pyunits.mol)\n",
+    "m.fs.pervap.energy_activation.fix()\n",
+    "\n",
+    "permeance_dict = {(0, 'Liq', 'water'): 5611320, (0, 'Liq', 'ethylene_glycol'): 22358.88}  # calculated from literature data\n",
+    "m.fs.pervap.permeance = Var(m.fs.time, m.fs.pervap.phase_list, m.fs.pervap.comp_list,\n",
+    "                            initialize=permeance_dict, units=pyunits.mol/pyunits.s/pyunits.m**2)\n",
+    "m.fs.pervap.permeance.fix()\n",
+    "\n",
+    "m.fs.pervap.area = Var(m.fs.time, initialize=6, units = pyunits.m**2)\n",
+    "m.fs.pervap.area.fix()\n",
+    "\n",
+    "cond_heat_dict = {(0, 'Liq', 'water'): 40.660e3, (0, 'Liq', 'ethylene_glycol'): 56.9e3}\n",
+    "m.fs.pervap.cond_heat = Var(m.fs.time, m.fs.pervap.phase_list, m.fs.pervap.comp_list,\n",
```

Comment:
I'm assuming that this is the latent heat of vaporization, but `cond_heat` is pretty cryptic. Ideally we'd use fully readable names.